### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- Include navbar
- Include style to navbar

## Jira Story

https://trello.com/c/VH8uBFya

## Screenshot

navbar

### CODE REVIEW

File: mock-up/index.html [hunk 1]
```diff
@@ -11,6 +11,11 @@
 </head>
 
 <body>
+  <div class="nav">
+    <img src="assets/wolox_logo.svg" alt="wolox" class="nav-logo">
+    <h3 class="nav-title">Developer Tools</h3>
+  </div>
+
```
Comment (FrankIglesias) [inline mock-up/index.html]: remove extra  line

File: mock-up/scss/colors.scss [hunk 2]
```diff
@@ -1 +1,3 @@
 $earls-green: #BED23C;
+$white: white;
```
Comment (FrankIglesias) [inline mock-up/scss/colors.scss]:
```suggestion
$white: #FFF;
```

File: mock-up/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 20px 0 $gray;
+  display: flex;
+  height: 70px;
+  justify-content: space-between;
+  padding: 20px;
+  position: sticky;
```
Comment (FrankIglesias) [inline mock-up/scss/navbar.scss:9]: absolute

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: mock-up/scss/navbar.scss (hunk 3 above)
Comment (joredudiaz) [reply mock-up/scss/navbar.scss:9]: if I put absolute the navbar stay in the top and when scroll the elements the whole page is scroll down (including the navbar)

File: mock-up/index.html [hunk 4]
```diff
@@ -11,11 +11,14 @@
 </head>
 
 <body>
+  <div class="nav">
+    <img src="assets/wolox_logo.svg" alt="wolox" class="nav-logo">
```
Comment (betoobernal) [inline mock-up/index.html]: add ``

**Review by betoobernal — APPROVED** [review]

**Pull request merged**